### PR TITLE
refactor: add number util

### PR DESCRIPTION
**Pull request by lissavxo**
Related to #405 

Description
---

Added handler with BigNumber logic in utils

Test plan
---
yarn build && yarn dev

### CODE REVIEW

**Review by chedieck — APPROVED** [review]

**Review by Klakurka — CHANGES_REQUESTED** [review]
How about adding some appropriate tests?

File: react/lib/tests/number.test.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import BigNumber from 'bignumber.js';
+import { zero, resolveNumber, isLessThanZero } from '../util/number'; // Adjust the import path
+
+describe('Number util functions', () => {
+  it('zero should be an instance of BigNumber and equal to 0', () => {
+    expect(zero).toBeInstanceOf(BigNumber);
+    expect(zero.isEqualTo(0)).toBe(true);
+  });
+
+  it('resolveNumber should convert various types to BigNumber', () => {
```
Comment (chedieck) [inline react/lib/tests/number.test.ts:10]: let's tests floats too (42.0, 42.5) and string floats ('42.0', '42.5'). Also, they shouldn't be in the same tests

File: react/lib/tests/number.test.ts [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    expect(resolveNumber('42')).toBeInstanceOf(BigNumber);
+    expect(resolveNumber('42').isEqualTo(42)).toBe(true);
+    
+    const bigNumberInstance = new BigNumber(42);
+    expect(resolveNumber(bigNumberInstance)).toBeInstanceOf(BigNumber);
+    expect(resolveNumber(bigNumberInstance).isEqualTo(42)).toBe(true);
+  });
+
+  it('isLessThanZero should correctly identify values less than zero', () => {
+    expect(isLessThanZero(-1)).toBe(false);
```
Comment (chedieck) [inline react/lib/tests/number.test.ts:23]: isLessThanZero(-1) returns false? why?

File: react/lib/util/number.ts [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import BigNumber from 'bignumber.js';
+
+export const zero = new BigNumber(0);
+
+export const resolveNumber = (value: BigNumber.Value) => {
+  return new BigNumber(value);
+};
+
+export const isLessThanZero = (value: BigNumber.Value) => {
```
Comment (chedieck) [inline react/lib/util/number.ts:9]: This function is implemented backwards. It's named `isLessThanZero` but it checks if `zeroIsLessThan`

**Review by chedieck — CHANGES_REQUESTED** [review]

**Review by chedieck — APPROVED** [review]

**Review by Klakurka — APPROVED** [review]

**Pull request merged**